aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -18,28 +18,27 @@ use move_vm_runtime::{
     config::VMConfig, move_vm::MoveVM, native_extensions::NativeContextExtensions,
 };
 use std::ops::Deref;
-use aptos_types::on_chain_config::Features;
+use aptos_types::on_chain_config::{FeatureFlag, Features};
 
 pub struct MoveVmExt {
     inner: MoveVM,
     chain_id: u8,
-    _features: Features,
+    features: Features,
 }
 
 impl MoveVmExt {
     pub fn new(
         native_gas_params: NativeGasParameters,
         abs_val_size_gas_params: AbstractValueSizeGasParameters,
         gas_feature_version: u64,
-        treat_friend_as_private: bool,
-        allow_binary_format_v6: bool,
         chain_id: u8,
+        #[allow(unused)]
```

Comment:
I don't think you need this now that you're using it?

Reply:
not used in in this PR...

Reply:
oh, this annotation went to the wrong place... fixing.